.changeset/dry-swans-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+"search": patch
+---
+
+Recreating webhooks feature has been removed from app
```

Comment:
Any more context "why" - eg. something like:
- "Now, recreating webhooks was removed, this means X no longer happens"
- "X no longer [issue it caused], that means recreating webhooks was removed"

or something similar

It's because you delete something but there is no info why you do this